store/sqlstore/store.go

```diff
@@ -403,6 +428,42 @@ func (ss *SqlStore) TotalMasterDbConnections() int {
 	return ss.GetMaster().Db.Stats().OpenConnections
 }
 
+// ReplicaLagAbs queries all the replica databases to get the absolute replica lag value
+// and updates the Prometheus metric with it.
+func (ss *SqlStore) ReplicaLagAbs() {
+	for i, item := range ss.settings.ReplicaLagSettings {
+		if *item.QueryAbsoluteLag == "" {
+			continue
+		}
+		var binDiff int
+		var node string
+		err := ss.replicaLagHandles[i].QueryRow(*item.QueryAbsoluteLag).Scan(&node, &binDiff)
+		if err != nil {
+			mlog.Warn("ReplicaLagAbs Query returned an error", mlog.Err(err))
+			return
+		}
+		ss.metrics.SetReplicaLagAbsolute(node, float64(binDiff))
```

Comment:
Is `ss.metrics` guaranteed to be non-nil?

Reply:
That's a good point. I will check.

Reply:
Yes, since this method is called from the metrics store, it will always be. I will add a comment to clarify that.